### DISCUSSION TITLE
How to restart simulation with Higher Order Polynomial basis

### DISCUSSION THREAD

**Discussion by garicano** [loganoz/horses3d#149] [Q&A]
Hello: 

I have computed a steady solution for the laminar flow around a circular cylinder (ReD=40, Ma=0.3) using polinomial order 3. 

I'm trying to restart the computation, but using polynomial order 5 (see attached control file). 

I receive the following error message:    
Error reading restart file: wrong dimension for element 1.
Element dimensions: 5 ,5 ,0.
Restart dimensions: 3 ,3 ,0.
Error in file HexMesh.f90, in line 3330.

Could you please help me with the control file configuration? Thanks!

I attach the solution file as well, in case it is useful for reproducing the error message.

[CylCirc.hsol.txt](https://github.com/loganoz/horses3d/files/11251028/CylCirc.hsol.txt)
[CylinderCircularSection.control.txt](https://github.com/loganoz/horses3d/files/11251019/CylinderCircularSection.control.txt)

**Comment by loganoz** [comment]
@garicano, control file does not support TABS. Can you check if there is a TAB in any of the lines?

**Reply by loganoz** [reply]
Additionally, as 2D mesh offset direction = z the "restart polorder file" should be used specifying 3 3 0 for every element.
